packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -577,9 +592,9 @@ public void onCreateView(Context initContext,
                                     .setUseDeveloperSupport(isDebug)
                                     .setJavaScriptExecutorFactory(new HermesExecutorFactory())
                                     .setInitialLifecycleState(LifecycleState.BEFORE_CREATE);
-        if (BuildConfig.SHOULD_ATTACH_JS_BUNDLE) {
+//        if (BuildConfig.SHOULD_ATTACH_JS_BUNDLE) {
```

Comment:
Hey @hypest - was this a temporary change?

Reply:
oops, never really meaning to commit that, that's a workaround to enable building the production WPAndroid (vanillaRelease) using local source mode. I will revert the change, thanks for catching it!

Reply:
Fixed with https://github.com/WordPress/gutenberg/commit/f918980309bf871417a558eb4be79af5f9bedd0d on the my PR, commit pending to be included in this PR too.

Reply:
Sounds good, thanks!